llvm/include/llvm/Demangle/ItaniumDemangle.h

```diff
@@ -27,6 +27,7 @@
 #include <cstdlib>
 #include <cstring>
 #include <numeric>
+#include <string>
```

Comment:
I think we don't need using `string` in this file - `StringView` should be enough. Please, remove `string` usage instead.
`assert((N == "16") && "Unknown FP type");` <- this should work.

Reply:
OK

Reply:
```suggestion
```